### PR TITLE
[pl_pl] & [de_de] Added translations

### PR DESCRIPTION
**Pull request by Kri5t0fK**
- added all new translations in [pl_pl]
- added missing line in [de_de]
- added `empty lines` in [pl_pl] & [de_de] so that line numbers are in line with [en_en]

### CODE REVIEW

File: assets/bingo/lang/de_de.json [hunk 1]
```diff
@@ -1,4 +1,9 @@
 {
+
+
+
+
+
```
Comment (NeunEinser) [inline assets/bingo/lang/de_de.json:6]: What's the padding for?

Comment (Kri5t0fK) [reply assets/bingo/lang/de_de.json:6]: For easier checking files side by side - with that I found [missing line](https://github.com/NeunEinser/bingo/pull/134/files#diff-1472095dcdb53ae3ffb36516720823c450b536f55248ef1a608d01ca3b97e37cR77) in [de_de]

Comment (NeunEinser) [reply assets/bingo/lang/de_de.json:6]: Ah so this is where technical strings are in en_us, so you have the same line numbers between both files?

**Review by NeunEinser — APPROVED** [review]

**Comment by NeunEinser** [comment]
Translation looking good, thank you 🙂

File: assets/bingo/lang/de_de.json (hunk 1 above)
Comment (Kri5t0fK) [reply assets/bingo/lang/de_de.json:6]: Yes, indeed ;)

**Pull request merged**